_data-prepper/getting-started.md

```diff
@@ -28,10 +28,39 @@ If you have special requirements that require you to build from source, or if yo
 
 ## 2. Configuring Data Prepper
 
-You must configure Data Prepper with a pipeline before running it. You'll modify the following files:
+When you run a Data Prepper instance, it requires two configuration files to run, and also allows you to use an optional Log4j 2 configuration file. See [Configuring Log4j]({{site.url}}{{site.baseurl}}/data-prepper/managing-data-prepper/configuring-log4j/) for more information on configuring Log4j in Data Prepper. The following table explains the purpose of each configuration file.
 
-* `data-prepper-config.yaml`
-* `pipelines.yaml`
+Option | Required | Description
+:--- | :--- | :--- |
+`pipelines.yaml` | Yes | This file describes which data pipelines to run, including sources, processors, and sinks. |
+`data-prepper-config.yaml` | Yes | This file contains Data Prepper server settings so that you can interact with exposed Data Prepper server APIs. |
+`log4j2-rolling.properties` | No | This file contains Log4j 2 configuration options and can be a JSON, YAML, XML, or a .properties file type. |
+
+For Data Prepper previous to version 2.0, the `.jar` file expects the pipeline configuration file path to be followed by the server configuration file path. See the following configuration path example.
```

Comment:
```suggestion
For Data Prepper versions below 2.0, the `.jar` file expects the pipeline configuration file path to be followed by the server configuration file path. See the following configuration path example:
```

Reply:
We shouldn't use "below" unless it's referring to a location or direction.

Reply:
~I think I remember Fanit using this language, too. Which tells me it's acceptable in the software business. This might be a call for Nate. For what it's worth, I would rewrite it ...~

~"For Data Prepper versions prior to 2.0, ..." Earlier than / later than could work.~

Scratch that.
Well, Microsoft style:
"Use or earlier or previous instead of or lower to refer to all versions of a product that precede a particular release if the statement is accurate for all preceding releases. For example, don't use Windows 10 or earlier unless the statement is accurate for Windows 1.0."

Apple:
"Earlier or later: Use earlier or later, rather than lower or higher or newer or older.
To use Handoff, you need iOS 8 or later."

AWS style is with the crickets on this one.